src/deployment/android.md

```diff
@@ -95,64 +95,66 @@ To find out the latest version, visit [Google Maven][].
 +<style name="NormalTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar">
 ```
 
-## Signing the app
+## Sign the app
 
-To publish on the Play Store, you need to give your app a digital
-signature. Use the following instructions to sign your app.
+To publish on the Play Store, you sign your app with a digital signature.
```

Comment:
I think "need" is important here as you need to do more than just sign the app to publish it. Also use "certificate" here to more closely align with the Android docs:

```suggestion
To publish on the Play Store, you need to
sign your app with a digital certificate.
```